modin/core/storage_formats/pandas/parsers.py

```diff
@@ -136,7 +137,7 @@ def find_common_type_cat(types):
         return find_common_type(list(types))
 
 
-class PandasParser(object):
+class PandasParser(object, metaclass=LoggerMetaClass):
```

Comment:
Given that the parser is more internal, should it really be logging with the loggermetaclass?

Reply:
I guess I've been logging every API call with the `LoggerMetaClass`. I don't think there is a need to distinguish between the pandas queries and the Modin internals. If folks want to just look at the pandas queries they are using, they have the script/notebook they are running them in and don't need Modin logging. Would you suggest that I rename `START::PANDAS-API::{classname}.{name}"` to `START::MODIN-API::{classname}.{name}"` in the LoggerMetaClass?

Reply:
I think it's fine the way it is.